### PR TITLE
use mixin to visualize keyboard focus

### PR DESCRIPTION
**Pull request by Kiarokh**
We use the same mixin in crm-components already.

## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/components/dock/dock-button/dock-button.scss [hunk 1]
```diff
@@ -44,6 +41,11 @@
         );
         box-shadow: var(--button-shadow-inset);
 
+        &:focus-visible {
+            outline: none;
+            box-shadow: var(--shadow-depth-8-focused);
+        }
+
```
Comment (LucyChyzhova) [inline src/components/dock/dock-button/dock-button.scss:48]: could we replace it with @mixins as well?

Comment (Kiarokh) [reply src/components/dock/dock-button/dock-button.scss:48]: ah it's further up. this is a repetition of the same code 😄 good find

Comment (Kiarokh) [reply src/components/dock/dock-button/dock-button.scss:45]: Also I noticed that we didn't have `disabled` on dock-button, so I removed the styles for it

Comment (Kiarokh) [reply src/components/dock/dock-button/dock-button.scss:45]: Good catch anyways!

**Review by LucyChyzhova — DISMISSED** [review]
Great optimization! 🤩

**Review by LucyChyzhova — APPROVED** [review]

**Pull request merged**